tests/Metadata/Driver/AnnotationDriverTest.php

```diff
@@ -53,13 +53,13 @@ public function testReadUploadableAnnotation(): void
         $reader
             ->expects(self::atLeastOnce())
             ->method('getPropertyAnnotation')
-            ->willReturnCallback(static fn (ReflectionProperty $property): ?UploadableField => 'file' === $property->getName() ? new UploadableField('dummy_file', 'fileName') : null);
+            ->willReturnCallback(static fn(ReflectionProperty $property): ?UploadableField => 'file' === $property->getName() ? new UploadableField('dummy_file', 'fileName') : null);
 
         $driver = new AnnotationDriver($reader, [$this->managerRegistry]);
         $metadata = $driver->loadMetadataForClass(new \ReflectionClass($entity));
 
         self::assertInstanceOf(ClassMetadata::class, $metadata);
-        self::assertObjectHasAttribute('fields', $metadata);
+        self::assertIsArray($metadata->fields);
```

Comment:
why?

Reply:
_assertObjectHasAttribute_ was deprecated:
https://github.com/sebastianbergmann/phpunit/issues/4601

Reply:
I think the import at the top of the file is invalid - `Metadata\ClassMetadata` should be replaced with `Vich\UploaderBundle\Metadata\ClassMetadata` - this way you can skip that assertion completely as `assertInstanceOf` will point to correct class now.